README.md

```diff
@@ -74,16 +74,30 @@ of what's defined in the configuration file.
 
 ## Compiling
 
-Download the repository and its dependencies:
+1. Install `idf.py`
```

Comment:
I wouldn't call it `idf.py`, that's just a single tool part of the SDK. Let's just leave it at `ESP-IDF`

Reply:
Done!